odoo/addons/base/models/ir_actions.py

```diff
@@ -385,6 +380,19 @@ def _get_readable_fields(self):
             "close_on_report_download",
         }
 
+    def _get_action_dict(self):
+        """ Override to return action content with detailed embedded actions data if available.
+
+            :return: A dict with updated action dictionary including embedded actions information.
+        """
+        result = super()._get_action_dict()
+        embedded_action_ids = result["embedded_action_ids"]
+        if embedded_action_ids:
```

Comment:
Should we use like this :thinking: 
```suggestion
        if embedded_action_ids := result["embedded_action_ids"]:
```

Reply:
Thanks!
I have updated code :+1: